app/views/events/show.html.erb

```diff
@@ -1,69 +1,77 @@
-<div class="container-fluid" style="padding-top: 20px; padding-left: 30px">
+<!-- Event title -->
+<div class="bg-image">
+  <br>
+    <h1 class="display-4" style="padding-left: 2%; color:white"><b><%= @event.name%></b></h1>
+    <h3 style="padding-left: 2%; color:white">
+      <%= @event.date.strftime('%A, %B %e, at %l:%M %p') %>
+    </h3>
+    <h2 style="padding-left: 2%">
+      <%= link_to 'Back', events_path, class: "btn btn-secondary text-white" if @event.date > Date.current %>
+      <%= link_to 'Back', past_events_path, class: "btn btn-secondary text-white" if @event.date < Date.current %>
+    </h2>
+</div>
+<!-- End event title -->
+<!-- Left side with image -->
+<div class="container-fluid" style="padding-left:5%; padding-top: 2%; padding-bottom: 1%; width: 30%; float: left;">
   <p>
     <%= image_tag @event.image.url, class: "event-show" %>
   </p>
-<h1><%= @event.name %></h1>
-<br>
-
-<p>
-  <strong>Description:</strong>
-  <%= @event.description %>
-</p>
-
-<p>
-  <strong>Date:</strong>
-  <%= @event.date %>
-</p>
-
-<p>
-  <strong>Location:</strong>
-  <%= @event.location %>
-</p>
-
-<p>
-  <strong>Tickets:</strong>
-  <%= @event.has_tickets %>
-</p>
-
-<p>
-  <strong>Tickets:</strong>
-  $<%= number_with_precision(@event.price, precision: 2) %>
-</p>
-
-<p>
-  <strong>Capacity:</strong>
-  <%= @event.capacity %>
-</p>
-
-<p>
-  <strong>Guests:</strong><br>
-  <% @guests.each do |guest| %>
-    <%= guest.firstname %> <%= guest.lastname %> <% if admin_signed_in? %> | <%= guest.email %> | <%= link_to 'Delete', event_guest_path(@event, guest), method: :delete, data: { confirm: 'Are you sure?' } %>
-    <% end %>
-    <br>
-  <% end %>
-</p>
-
-<p>
-  <strong>Share on Facebook</strong>
```

Comment:
this link is broken- check my branch if you wanna see how to fix. Also I think we should probably use custom logos for all social media links so we have some type of consistency.